styles.css

```diff
@@ -0,0 +1,75 @@
+/********************************************* HEADER *******************************************************************/
+.main-header-container {
+    color: white;
+    background-color: #f72210;
+}
+
+.top-logo {
+    filter: invert(1);
+}
+
+.mobile-logo {
+    width: 132px;
+    height: 20px;
+}
+
+.main-header-right-align {
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 700;
+}
+
+.login {
+    text-decoration: none;
+    color: white;
+}
```

Comment:
Avoid using custom styling where bootstrap styling is possible. Example in the above, you can `text-decoration-none` and `text-white` bootstrap classes. Please check the rest of your styling as well.